electron/legendary/games.ts

```diff
@@ -200,6 +201,51 @@ class LegendaryGame extends Game {
     }
   }
 
+  /**
+   * Adds a desktop shortcut to $HOME/Desktop and to /usr/share/applications
+   * so that the game can be opened from the start menu and the desktop folder.
+   * Both can be disabled with enableDesktopShortcutsOnDesktop and enableDesktopShortcutsOnStartMenu
+   * @async
+   * @public
+   */
+  public async addDesktopShortcut() {
```

Comment:
Would be nice if you can write test for this function. Wouldn't be that hard. Atleast not harder then mocking process.exit.

Reply:
I don't know...We'd need to mock this.getGameInfo first. Don't we?

Reply:
I will try to do so.